libs/ReinforcerKit/source/ReinforcerKit.cpp

```diff
@@ -17,9 +17,6 @@ void ReinforcerKit::setDefaultReinforcer(const Reinforcer &reinforcer)
 void ReinforcerKit::play(const Reinforcer &reinforcer)
 {
 	switch (reinforcer) {
-		case Reinforcer::Rainbow:
-			playRainbow();
-			break;
```

Comment:
Why remove this reinforcer?

Reply:
I agree with @YannLocatelli -- the fact that the default is Rainbow does not mean we can remove the case.

you could have but I prefer to have Rainbow at the top

```
		case Reinforcer::Rainbow:
		default:
			playRainbow();
			break;
```